frontend/src/components/UpcomingReminder/UpcomingReminder.jsx

```diff
@@ -48,10 +48,10 @@ const UpcomingReminder = () => {
 
     useEffect(() => {
         dispatch(fetchUserReminders(currentUser._id))
-    }, [dispatch, currentUser._id])
+    }, [dispatch, currentUser._id], upcomingReminders)
```

Comment:
I've never seen this syntax before, what's the 3rd argument to useEffect do?

Reply:
I think it was a typo. I will erase that and push it

Reply:
Was it supposed to be like this?
```suggestion
    }, [dispatch, currentUser._id, upcomingReminders])
```

Reply:
you can take upcomingReminders, so only dispatch, currentUser._id in the dependency array

Reply:
Gotcha, will do.